src/content/tutorials/en/setting-up-your-environment.mdx

```diff
@@ -73,7 +73,7 @@ If you are new to p5.js, start with the [p5.js Web Editor](https://editor.p5js.o
   - Create and confirm a password. 
   - Click the “Sign Up” button.
 
-![An arrow pointing to the highlighted User Name, Emaill, Password, and Confirm Password fields on the Sign Up page.](../images/introduction/p5_editor_sign_up_page.png)
+![An arrow to the highlighted fields on Sign-Up page.](../images/introduction/p5_editor_sign_up_page.png)
```

Comment:
E.g., "An arrow pointing to the highlighted fields on the Sign Up page, as described above."